src/Angor/Client/Pages/Unfunded.razor

```diff
@@ -613,12 +613,12 @@
     {
         try
         {
-            var key = DerivationOperations.DeriveFounderRecoveryPrivateKey(words, FounderProject.ProjectIndex);
+            var key = DerivationOperations.DeriveFounderRecoveryPrivateKey(words, FounderProject.ProjectInfo.FounderKey);
```

Comment:
perhaps we should keep the UPI as a property on the project info?

Reply:
Nah it just complicates things and that parameter is only ever used for derivation